internal/controller/clientprofile_controller.go

```diff
@@ -130,6 +130,10 @@ func (r *ClientProfileReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 
 func (r *ClientProfileReconcile) reconcile() error {
 	if err := r.loadAndValidate(); err != nil {
+		if client.IgnoreNotFound(err) == nil {
+			r.log.Info("Resources for reconciling clientprofile is not found. Object(s) might have been deleted.")
```

Comment:
Let's be more specific in the log message:
```suggestion
			r.log.Info("Client profile resource does not exists anymore, skipping reconcile")
```

Reply:
done.